kodadot/nft-gallery
# Too Soon Junior

- docs(CONTRIBUTING): update how to receive rewards
- wip
- ✨ remove empty rmrk collecntions
- 🧪 testing mobile wallet preview
- 🧪 testing mobile wallet preview accounts
- 🧪 bad code mobile accounts
- 🧪 remove unused code
- fix profile tabs
- add fallback user name
- refactor display name
- feat: add first version of style guide
- code refactor
- #2738 Hiding the burned nfts
- feat: add best practice code for data fetching
- fallback nft meta is null
- justify collection stats
- docs(CONTRIBUTING): new button for sending reward
- feat: add holdings by user
- custom label name
- fix checkbox selector
- display collection name in detail page
- add collections collected and created by user
- tooltip rename
- display holding tab forever
- style remove scoped
- fix: switch button broken at some video nft on mobile
- add types
- testing auto assign
- auto-assign.yml
- wrong dir
- auto-assign.yml
- according to #2730
- maybe needs more space
- #2767 Link from series insight to collection chart fix
- feat: Navbar breakpoints
- change id
- adding in salevents query
- fix nav missing on mobile
- sorted through CODEOWNERS
- chore: add general guidelines
- :nail_care: :memo: styling the STYLE_GUIDE
- fix: justify nav-tab hiding order
- responsive search bar and hide nav-bar
- fix: fallback metadata
- add image string
- nav-bar display on different width
- ⚒️ mobile landing
- feat nfts event query
- #2774 Limit in caraousel
- refactor query structor
- fix: nav button top align
- #2771 Bug in hover card "minting ago"
- #2771 adding last bought support in popover
- feat:  infinite scroll on collection page
- adding burned filter
- :bug: fix estimation for the support fees
- :bug: use or instead of nullish coalescing
- refactor code
- feat link holder user to hodings tab
- fix incorrect route
- fix tab from url
- fix typeScript warning
- code refactor
- fix query error
- cut long name
- key change
- hotfix ts error
- feat: add hover artist card to holdings
- refactor: remove any type on holder component
- fix hover style on card
- mobile menu ui
- feat: inject consola plugin
- refactor: from console to consola
- chore: correct comment
- feat: mobile navbar ui
- css justify
- ⚒️ remove unused css + fix mobile navbar buttons
- ⚒️ remove unused css + small refactoring
- fix: renovate minor bumps
- removed schedule temporary
- ⚒️ remove flex card searchbar from navbar
- chore: activate nitro again
- #2489 Sticky header broken at Series
- feat: Add Flippers to Collection
- fix search is broken
- add percentage column to flipper
- extend event query
- delete unused query
- justify border style
- aero effect
- Adjust renovate with minor/patch only (#2809)
- refactor border variables
- Navbar changes, common component
- refactor fallback
- search changes
- route changes everywhere
- visual updates
- removing comment
- code refactor
- other chain changes
- feat infinite scroll on collection page
- add ts type
- Update STYLE_GUIDE.md
- code refactor
- chore: use modern build
- refactor: use modern format
- fix: lazyimports and fixed image sizes
- Update components/rmrk/Collection/List/CollectionList.vue
- Update components/rmrk/ExploreLayout/ExploreLayout.vue
- minor changes
- ⚒️ rewrite query subsquid
- redirect middleware
- show gallery first
- reverting order, minor path name change
- removing mixin
- redirect fixes, refactor
- netlify redirect
- test netlify
- redirect logic change
- change order back to gallery
- route minor change
- Update pages/errorcollection.vue
- Update components/rmrk/ExploreLayout/ExploreLayout.vue
- Update components/rmrk/ExploreLayout/ExploreLayout.vue
- error fix
- startswith check
- #2807 Remove pinterest and iframe from sharing button, ux refactor
- ⚒️ subsquid query + fix westend
- Update components/rmrk/Gallery/GalleryCardList.vue
- Update components/shared/LinkResolver.vue
- add average gain on flippers
- comment remove
- fix: added fallback for safari pre big sur
- [skip netlify] Update Bumped minor version of packages
- add hiring.md
- add rewards.md
- gh markdown links differently
- links between docs
- links between docs
- probably done
- probably satisfied with new readme reorg
- repography aligned <3
- Revert "[skip netlify] Update Bumped minor version of packages (patch)"
- move fallbackMetaByNftEvent
- bump patch vitest
- revert
- pin nuxt/bridge
- move to carousel
- check dist folder
- pin nuxt-edge version and check folder in github actions
- add types in utils/carousel
- compatible mobile
- use consola
- refactor: using nuxt helper to detect network connection
- #2838 Link in /collection/{id} are broken on beta
- update hiring
- #2840 Post explore unify fixes
- minor issue fixes,search term css
- added collection tab first
- add query after search
- Update components/rmrk/ExploreLayout/ExploreLayout.vue
- Update components/rmrk/ExploreLayout/ExploreLayout.vue
- margin issue
- fix search-navbar-container-mobile input bg
- ⚒️ collections listed filter
- change in logic

**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type


### What's new?


### Before submitting Pull Request, please make sure:


### Optional


### Had issue bounty label?


### Community participation


### Screenshot
